<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePartitionDialog</class>
 <widget class="QDialog" name="CreatePartitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a Partition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="PartResizerWidget" name="partResizerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>59</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Size:</string>
       </property>
       <property name="buddy">
        <cstring>sizeSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sizeSpinBox">
       <property name="suffix">
        <string> MiB</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="partitionTypeLabel">
       <property name="text">
        <string>Partition &amp;Type:</string>
       </property>
       <property name="buddy">
        <cstring>primaryRadioButton</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="primaryRadioButton">
         <property name="text">
          <string>&amp;Primary</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="extendedRadioButton">
         <property name="text">
          <string>&amp;Extended</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fixedPartitionLabel">
         <property name="text">
          <string notr="true">[fixed-partition-label]</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;File System:</string>
       </property>
       <property name="buddy">
        <cstring>fsComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="fsComboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="EncryptWidget" name="encryptWidget" native="true"/>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="mountPointLabel">
       <property name="text">
        <string>&amp;Mount Point:</string>
       </property>
       <property name="buddy">
        <cstring>mountPointComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="mountPointComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelMountPoint">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Flags:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QListWidget" name="m_listFlags">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PartResizerWidget</class>
   <extends>QWidget</extends>
   <header location="global">kpmcore/gui/partresizerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EncryptWidget</class>
   <extends>QWidget</extends>
   <header>gui/EncryptWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>primaryRadioButton</tabstop>
  <tabstop>fsComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreatePartitionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreatePartitionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>extendedRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>fsComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>extendedRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
